src/components/DeeplinkWrapper/index.website.js

```diff
@@ -0,0 +1,157 @@
+import _ from 'underscore';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import React, {PureComponent} from 'react';
+import deeplinkRoutes from './deeplinkRoutes';
+import FullScreenLoadingIndicator from '../FullscreenLoadingIndicator';
+import TextLink from '../TextLink';
+import * as Illustrations from '../Icon/Illustrations';
+import withLocalize, {withLocalizePropTypes} from '../withLocalize';
+import Text from '../Text';
+import styles from '../../styles/styles';
+import CONST from '../../CONST';
+import CONFIG from '../../CONFIG';
+import Icon from '../Icon';
+import * as Expensicons from '../Icon/Expensicons';
+import colors from '../../styles/colors';
+import * as Browser from '../../libs/Browser';
+
+const propTypes = {
+    /** Children to render. */
+    children: PropTypes.node.isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+class DeeplinkWrapper extends PureComponent {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            appInstallationCheckStatus: this.isMacOSWeb() ? 'checking' : 'not-installed',
```

Comment:
Use consts for `checking` and `not-installed`?

Reply:
I'm, fine with both as its not used any where else.

Reply:
Done